vi/lessons/advanced/erlang.md

```diff
@@ -1,28 +1,28 @@
 ---
 layout: page
-title: Erlang Interoperability
+title: Tương tác với Erlang
 category: advanced
 order: 1
 lang: vi
 ---
 
-One of the added benefits to building on top of the ErlangVM is the plethora of existing libraries available to us.  Interoperability allows us to leverage those libraries and the Erlang standard lib from our Elixir code.  In this lesson we'll look at how to access functionality in the standard lib along with third-party Erlang packages.
+Một trong những lợi ích có được khi xây dựng dựa trên ErlangVM (máy ảo Erlang) là sự phong phú của các thư viện có sẵn. Khả năng tương tác với Erlang cho phép ta sử dụng những thư viện này cũng như thư viện chuẩn của Erlang khi viết code bằng Elixir. Ở bài này, chúng ta sẽ xem xét việc truy cập những tính năng của thư viện chuẩn cũng như thư viện bên thứ ba của Erlang.
 
-## Table of Contents
+## Mục lục
 
-- [Standard Library](#standard-library)
-- [Erlang Packages](#erlang-packages)
-- [Notable Differences](#notable-differences)
-  - [Atoms](#atoms)
-  - [Strings](#strings)
-  - [Variables](#variables)
+- [Thư viện chuẩn](#standard-library)
+- [Erlang Packages (thư viện bên thứ ba)](#erlang-packages)
```

Comment:
please remove `(thư viện bên thứ ba)` here to keep the header short and concise.